3.3 release prep

osfameron opened the pull request:
* merge 3.2 branch work
* update Start Using page
* bump server to 7.1
* Update compatibility

BKaneAtWork committed:
typo: missing closing parenthesis

osfameron committed:
Update antora links/includes to 7.1

  git grep -l \\b7\\.0@ | xargs sd '7\.0@' '7.1@'

osfameron committed:
Update server 7.0 mentions

Manual pass

daschl committed:
3.2.7 Release Notes

osfameron committed:
Update compatibility for 3.3 / 7.1

osfameron committed:
Update pom.xml

osfameron committed:
3.3 Start using guide update

Capella example aligned with normal getting started.

osfameron committed:
Gardening

maria-robobug commented on modules/devguide/examples/CloudConnect.java:
```diff
@@ -14,97 +14,82 @@
+  public static void main(String... args) {
+    // tag::connect[]
+    // Configure TLS
+    SecurityConfig.Builder securityConfig = SecurityConfig
```
Do we still need this? Thought this was automatic now for Capella connections?

maria-robobug replied:
Works for me in Go without a Security config, that's why I ask. 
Not sure if the same or different for Java 😕 .

osfameron replied:
Don't know - I removed the Certificate wrangling, but I thought you still had to ensure it was TLS? @daschl to confirm?

osfameron replied:
Ah ok. Lemme try without, even better if so...

osfameron replied:
`An exception occured while executing the Java class. The DNS SRV record for 'cb.njg8j7mwqnvwjqah.cloud.couchbase.com' indicates TLS must be used when connecting to this cluster. Please enable TLS by setting the 'security.enableTLS' client setting to true. If the Cluster does not use a shared ClusterEnvironment, an alternative way to enable TLS is to prefix the connection string with "couchbases://" (note the final 's')`

osfameron replied:
Yup, works great with `couchbases://` in connection string...
That's definitely the way to go for Hello World example, thanks!

osfameron replied:
(also nice error message Michael & co!)

osfameron committed:
Remove excess guff from Capella example

Extra faffing not needed, hurrah!
NB: CloudConnect.java and StartUsing.java are now functionally
identical.

maria-robobug reviewed (approved):
👍
